src/js/components/SupportBanner/SupportBanner.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { CImg } from '@coreui/react';
+import SupportService from '../config/supporter-config';
+
+const SupportBanner = () => (
+  <div className="support-banner p-0 h-auto">
+    <div className="supporters container w-100 p-4 align-items-center d-flex flex-wrap">
+      <SupportItems />
+    </div>
+  </div>
+);
+
+export const SupportItems = () => SupportService
+  .map((item) => (
```

Comment:
```suggestion
export const SupportItems = () => SupportService.map((item) => (
```

Is there a reason for dropping the line?